From 76183805b1254bb8e3df6197cecb323f36441df7 Mon Sep 17 00:00:00 2001
From: Steve Kondik <steve@cyngn.com>
Date: Thu, 11 Aug 2016 14:32:10 -0700
Subject: [PATCH 018/249] Add HexoLibre to manifest!

 * Now with 100% more OSS goodness
 * Dave Kover is a god amongst men
 * Just sayin'

Change-Id: Ia2137938679e90883c38a056f3a2d74f8fc117a9
---
 default.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/default.xml b/default.xml
index d9bc36f..7b617a6 100644
--- a/default.xml
+++ b/default.xml
@@ -270,6 +270,7 @@
   <project path="packages/services/Telecomm" name="CyanogenMod/android_packages_services_Telecomm" groups="pdk-cw-fs,pdk-fs" />
   <project path="packages/services/Telephony" name="CyanogenMod/android_packages_services_Telephony" groups="pdk-cw-fs,pdk-fs" />
   <project path="packages/services/ThemeManagerService" name="CyanogenMod/android_packages_services_ThemeManagerService" groups="pdk-cw-fs,pdk-fs" />
+  <project path="packages/themes/HexoLibre" name="CyanogenMod/android_packages_themes_HexoLibre" groups="themes" />
   <project path="packages/wallpapers/Basic" name="CyanogenMod/android_packages_wallpapers_Basic" groups="pdk-fs" />
   <project path="packages/wallpapers/Galaxy4" name="CyanogenMod/android_packages_wallpapers_Galaxy4" groups="pdk-fs" />
   <project path="packages/wallpapers/HoloSpiral" name="CyanogenMod/android_packages_wallpapers_HoloSpiral" groups="pdk-fs" />
-- 
2.17.1

